### PR TITLE
  Remove MqueueAdapter

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Mqueue Adapter - it's unused code in component HMIMessageHandler, so I deleted it.

### CODE REVIEW

**Comment by LitvinenkoIra** [comment]
@LuxoftAKutsan @anosach-luxoft @AGaliuzov @dtrunov  please review.

**Comment by LuxoftAKutsan** [comment]
@LitvinenkoIra please squash commits in PR

**Comment by LitvinenkoIra** [comment]
@LuxoftAKutsan Done.

**Comment by LitvinenkoIra** [comment]
Rebuild required

File: src/components/include/test/hmi_message_handler/mock_hmi_message_handler.h [hunk 1]
```diff
@@ -1,54 +0,0 @@
-/*
```
Comment (anosach-luxoft) [inline src/components/include/test/hmi_message_handler/mock_hmi_message_handler.h]: @LitvinenkoIra why are you going to remove this file?

Comment (LitvinenkoIra) [inline src/components/include/test/hmi_message_handler/mock_hmi_message_handler.h]: @anosach-luxoft Because we used it only in mqueue_adapter_test.

Comment (anosach-luxoft) [inline src/components/include/test/hmi_message_handler/mock_hmi_message_handler.h]: @LitvinenkoIra ok

**Pull request merged**